### ISSUE TITLE
Move "not interest" menu.

### ISSUE THREAD

**Issue by bartoszek** [yuliskov/SmartTube#1234]
I've experienced an issue when the app is getting sluggish and unintentionally I chose "not interest" instead of "add to playlist" from the context menu.

Could we perhaps move the "not interest" option to the bottom of the context menu, please?

**Comment by stevensmedia** [comment]
Please no. The home screen is so full of trash, moving Not Interested down would be a horrible experience for me.

**Comment by bartoszek** [comment]
@stevensmedia Means you are actually using this option regularly?

**Comment by elle8che** [comment]
I came to open this as a request and found this thread. I agree on moving the Not Interested menu item down. It is too easy to click not interested and lose a video when you were hoping to click a "Add to.." menu item. This NI menu item is relatively recent, and I was very used to the previous menu arrangement where "Add to..." was the uppermost menu item. I have yet to get used to this switch.

Also, I have a MUCH greater need to add videos to a playlist than the need to select ones I am not interested in. Typically, I use the "Add to..." menu a good 10x more often. If I have so many videos that I am not interested in, I believe that's my sign to re-train YT to show me what I DO want to see --- click more "yes" videos and fewer "no" and questionable videos, remove disliked video types from my history, or even watch certain videos in incognito mode (when I'm on desktop). Minus a few misses (one or two that never seem to vanish no matter how hard I try), my YT viewing experience is nicely curated.

Lastly, we use YT and STN in unique ways, but this one thing may be helpful to others. Currently, if I right click on video, the menu order is 
1. Not Interested 
2. Add to (whatever last playlist I added/removed from) 
3. Add/Remove from playlist. 
4. ...and so on
 
I would greatly appreciate adding **a permanent "Add to Watch Later" as a menu item** and preferably in slot 1. It would then look like 
1. Add to Watch Later
2. Add to (whatever last playlist I added/removed from)
3. Not Interested
5. ...and so on

Here's why: The current menu setup requires only 2 clicks to get to (and execute) Not Interested. To get to (and execute) "Add to Watch Later" -- which is in a submenu -- takes 5 clicks plus 2 more to exit the menu. Is there a big difference between 2 clicks and 7 clicks? YES! There are only 6 items in the main right click menu, and (at least on my screen -- Fire stick) there is ample room left for another menu item. "Add to Watch Later" would fit perfectly on this page, avoiding extra clicks, wearing down of the remote and batteries, general frustration, etc. 

Thank you for this great app -- I've been using it since SmartYouTube and find it to be incredible helpful, and yuliskov & support to be attentive and considerate.

**Comment by elle8che** [comment]
> Please no. The home screen is so full of trash, moving Not Interested down would be a horrible experience for me.

Consider clearing your history, cache, everything, or even starting a new account so you can train YT from a clean'ish slate. "Training" is relative as YT is it's own master, but we don't have some control over what content is presented. If you create a new account, you can add it alongside your current account, then select your original account any time you'd like to go back to it... for old times sake.

**Comment by DiscoFever78** [comment]
Yes, PLEASE move "not interested". I've lost so many videos that I wanted to add to my watch list but had deleted by this evil option that shows up intermittently when it wants to.
